templates/.eslintrc.json

```diff
@@ -21,14 +22,16 @@
   },
   "extends": [
     "airbnb",
+    "airbnb/hooks",
     "plugin:jest/recommended",
     "plugin:prettier/recommended"
   ],
   "plugins": [
     "jest",
-    "prettier"
+    "prettier",
+    "simple-import-sort"
```

Comment:
If we add hooks, do we also need to add the react plugin here? Or is that already taken care of with extending the airbnb config?

Reply:
As far as I can tell, this is taken care of by the `airbnb` config:

- https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v18.1.0/packages/eslint-config-airbnb/index.js#L4
- https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v18.1.0/packages/eslint-config-airbnb/rules/react.js#L7-L9